src/main/java/pw/kaboom/commandspy/Main.java

```diff
@@ -69,7 +69,7 @@ void onPlayerCommandPreprocess(final PlayerCommandPreprocessEvent event) {
 		for (String uuidString : config.getKeys(false)) {
```

Comment:
Thanks! Could you move the check outside the loop?
```suggestion
		if (event.isCancelled()) {
			return;
		}

		for (String uuidString : config.getKeys(false)) {
```

Reply:
Sure!